OSDOCS:13467: Creating new tutorial book and updating CLI tutorial (#90215)
openshift/openshift-docs: modules/getting-started-cli-scaling-app.adoc

// Module included in the following assemblies:
//
// * tutorials/dev-app-cli.adoc

:_mod-docs-content-type: PROCEDURE
[id="getting-started-cli-scaling-app_{context}"]
= Scaling up the deployment

> In Kubernetes, a `Deployment` object defines how an application deploys. In most cases, users use `Pod`, `Service`, `ReplicaSet`, and `Deployment` resources together. In most cases, {product-title} creates the resources for you.

Reviewer: ```suggestion
In Kubernetes, a `Deployment` object defines how an application deploys. In most cases, you use the `Pod`, `Service`, `ReplicaSet`, and `Deployment` resources together, and {product-title} creates the resources for you.
```

When you deploy the `parksmap` image, a deployment resource is created. In this example, only one pod is deployed. You might want to scale up your application to keep up with user demand or to ensure that your application is always running even if one pod is down.

> The following procedure scales the `parksmap` deployment to use two instances.

Reviewer: Consider removing this line since the first line of the Procedure states that you're scaling the deployment to use two instances.

parksmap-5f9579955-6sng8   1/1     Running   0          7m39s
parksmap-5f9579955-8tgft   1/1     Running   0          24s
----
+

> Verify that there are two `parksmap` pods listed.

Reviewer: ```suggestion
Verify that two `parksmap` pods are listed.
```

[TIP]
====

> You can scale your deployment back down to one pod instance by passing in `1` to the `--replicas` option:

Reviewer: ```suggestion
To scale your deployment back down to one pod instance, pass in `1` to the `--replicas` option:
```
